examples/test_feature_example_group_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe "test feature example group" do
+  feature "test.feature"
```

Comment:
oh yea, i realise that, i meant this particular thing `feature 'test.feature'` is very similar to turnip `steps_for`